gems/quilt_rails/lib/quilt_rails/react_renderable.rb

```diff
@@ -39,5 +41,11 @@ def initialize(url)
         super "Errno::ECONNREFUSED: Waiting for React server to boot up. If this error presists verify that @shopify/react-server is configured on #{url}"
       end
     end
+
+    class DoNotIntegrationTestError < StandardError
+      def initialize
+        super "Do not try to use Rails integration tests on your quilt_rails app. Instead use Jest and @shopify/react-testing to test your react application directly."
```

Comment:
I'd try to be less DO AS WE SAY:

```suggestion
        super "quilt_rails does not support Rails integration tests. Instead, use Jest and @shopify/react-testing to test your react application components in isolation."
```